### ISSUE TITLE
Other Gifts?

### ISSUE THREAD

**Issue by scohn** [Monika-After-Story/MonikaModDev#3151]
First, just wanna say I always love it when a new update comes out, you guys are so talented to be able to make something like this. 

Since you can give Monika gifts on days not holidays, is it possible to give gifts other than her birthday ones. For instance, I think I remember being able to give her roses on Valentines Day, so could we do that? If so, then what's the exact name for the file?

**Comment by HZhunter124** [comment]
Hope there will have chocolate as a gift because monika seems to like chocolate very much

**Comment by AsuraShun** [comment]
yeah, and probably flowers since I don't know how I can actually give her the roses from valentine

**Comment by scohn** [comment]
What about books, as a gift? Maybe some titles she's talked about but may not have, like Flowers for Algernon. If she's already got it, then something else she may like. She does love reading so it'd be really thoughtful to give that.

**Comment by ThePotatoGuy** [comment]
this was implemented in #3357
